docs/cookbooks/harness.rst

```diff
@@ -650,6 +669,22 @@ datafile.
              cli: vty
              yang: yang
 
+The custom connections are supported in mapping datafile, by the new schema:
```

Comment:
in line with other comments. custom -> multiple.  Not sure if we need 'new schema'

Reply:
updated custom to multiple connections.